src/java_binding/src/lib.rs

```diff
@@ -719,3 +719,19 @@ pub extern "system" fn Java_com_risingwave_java_binding_Binding_rowClose<'a>(
 ) {
     pointer.drop()
 }
+
+#[cfg(test)]
+mod tests {
+    use risingwave_common::types::DataType;
+    use risingwave_expr::vector_op::cast::literal_parsing;
+
+    /// make sure that the [`ScalarRefImpl::Int64`] received by
+    /// [`Java_com_risingwave_java_binding_Binding_rowGetTimestampValue`]
+    /// is of type [`DataType::Timestamptz`] stored in microseconds
+    #[test]
+    fn test_timestampz_to_interval() {
```

Comment:
I don't think this test is related to `interval`, please rename it.

Reply:
😅